sample/test.py

```diff
@@ -31,11 +26,20 @@ def run(self):
     def __init__(self):
         logging.basicConfig()
         self.logger = logging.getLogger(__name__)
-        #signal.signal(signal.SIGINT, signal.SIG_IGN)
         logging.getLogger('mesoshttp').setLevel(logging.DEBUG)
         self.driver = None
-        self.client = MesosClient(mesos_urls=['http://127.0.0.1:5050'])
-        #self.client = MesosClient(mesos_urls=['zk://127.0.0.1:2181/mesos'])
+        # By default use ZK for master detection
+        self.client = MesosClient(mesos_urls=['zk://leader.mesos:2181/mesos'])
+        secret = os.getenv('SERVICE_SECRET')
+        if secret:
+            # We are in Enterprise mode and using service account
+            # so we can rely on DCOS AdminRouter getting us to the Master as we
+            # will have an authentication token
+            self.client = MesosClient(mesos_urls=['https://leader.mesos/mesos'])
```

Comment:
same comment for dns

Reply:
In this case I'd disagree. Since service account and secrets are only going to be in DCOS, we can count on Mesos DNS being available.

Reply:
as suggested above with your explanation, this is fine for me